src/quo2/components/reactions/reaction.cljs

```diff
@@ -18,7 +18,7 @@
       {:size  20
        :color (if dark?
```

Comment:
We should get theme from the theme context provider in these components - is it possible to add as these changes?

Ícaros docs pr should describe how it works or you can check the button component

Reply:
Same for other uses of theme and colors/theme-color function.

colors/theme-color requires a 3rd parameter of 'theme'. While we migrate everything to this approach this param will be optional but eventually it will be mandatory to pass.

Reply:
Yep looks good!

Also theme-colors is used in src/status_im2/contexts/chat/composer/view.cljs so should probably do similar there.

Reply:
mmh, is a bit unhappy about it if I try the same (invalid hook call), would it be ok if I leave it as it is for this PR, as I think it might require a bit more shuffling around? Otherwise happy to do it

Reply:
Yeah of course, leave if it's causing hassle. We'll eventually get around to these files and make the adjustments for these. 👍